tests/SimpleSiteMap.Tests/ConvertToSiteMapTests.cs

```diff
@@ -9,33 +9,48 @@ namespace SimpleSiteMap.Tests
 {
     public class ConvertToSiteMapTests
     {
+        private static readonly DateTime StartDate = DateTime.SpecifyKind(new DateTime(2014, 11, 21, 18, 58, 00), DateTimeKind.Utc);
+        
+        private static readonly SitemapService SitemapService = new SitemapService();
+        
         [Fact]
         public void GivenDataWith10Items_ConvertToSiteMap_CreatesASitemapResult()
         {
             // Arrange.
-            var startDate = DateTime.SpecifyKind(new DateTime(2014, 11, 21, 18, 58, 00), DateTimeKind.Utc);
-            var data = SimpleSiteMapHelpers.CreateFakeSitemapNodes(100, startDate).ToPartition(10);
-            var siteMapService = new SitemapService();
+            var data = SimpleSiteMapHelpers.CreateFakeSitemapNodes(100, StartDate).ToPartition(10);

```

Comment:
Ah! that reminds me - i need to remove this extension and leverage MoreLinq, for this in the tests. Ta!!!